specs/UI/Snomed.spec

```diff
@@ -9,13 +9,13 @@ tags: snomed, ui
 * Logout and Login to Bahmni snomed location "General Ward" as a "doctor"
 * Open clinical tab
 * Open "Diagnoses" Tab
-* Doctor adds random snomed diagnosis from the file "consultation/diagnosis/disease" which is not present in Openmrs Database using "name" search
+* Doctor adds random snomed diagnosis with snomed code "404684003" which is not present in Openmrs Database using "name" search
```

Comment:
Why this code is hard coded?

Reply:
because this function is re-used that is why passing the snomed code here and later I'm concatenating with snomed url and hitting the api

Reply:
404684003 - what is this code in snomed ?

Reply:
fixed